aspnetcore/test/debug.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Debug ASP.NET Core apps
+author: rick-anderson
+description: Debug ASP.NET Core apps
+ms.author: riande
+ms.custom: mvc
+ms.date: 12/15/2019
+uid: test/debug
+---
+
+# Debug ASP.NET Core apps
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT) 
+
+The following links provide information on debugging ASP.NET Core apps.
+
+* [Debugging ASP Core on Linux](https://devblogs.microsoft.com/premier-developer/debugging-asp-core-on-linux-with-visual-studio-2017/)
+* [Debugging .NET Core on Unix over SSH](https://devblogs.microsoft.com/devops/debugging-net-core-on-unix-over-ssh/)
+* [Quickstart: Debug ASP.NET with the Visual Studio debugger](/visualstudio/debugger/quickstart-debug-aspnet)
+* See [this GitHub issue](https://github.com/aspnet/AspNetCore.Docs/issues/2960) for more debugging information.
```

Comment:
See this works fine for screen readers. The GitHub title might change. It's not good for SEO but we don't care about that. I'll probably change the issue.

Reply:
In context, perhaps they make more sense, and that's the case here (i.e., "for more debugging information" and in this section). Sometimes tho (my understanding is that) screen readers can present the links of a topic as an alphabetical list that people who use screen readers find very helpful to get to content that they know is linked *somewhere* on the page, so any links with "see this" language aren't helpful, and even the "for more debugging information" would be useless in that scenario. On a broader point at least as a consumer of web content, I like to have some indication about what something is linking to before I venture a click.

Perhaps more important, there's no information at this link destination. Just more links.

Reply:
The links are information. Links to debugging. Links I can't add to this doc. And that issue can grow.

Reply:
I was just clarifying my point given that I won't *shut up* about it. :smile: lol I've never been asked to use ambiguous link text, so I'm cool.

Since you want to keep this and *Troubleshoot* separate (i.e., don't merge into an *Overview* of the node), I only really have feedback on that extra line :point_up: and the date ☝️, which doesn't seem to be a problem any longer ... future dates used to end in :cry: tears around here, but it looks like they changed that at some point.